zulipterminal/ui_tools/boxes.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, view: Any) -> None:
         self.view = view
         self.msg_edit_id = None  # type: Optional[int]
         self.is_in_typeahead_mode = False
+        self.subscriber_ids = []  # type: List[int]
```

Comment:
I'm not sure, when you say 'recipient' would you associate it to large members of a stream?
'subscribers' isn't the right fit for private messages either though, since you can't choose to subscribe.

Reply:
Sounds good.